app/controllers/yachts_controller.rb

```diff
@@ -10,11 +10,9 @@ def new
   end
 
   def create
-    @yacht = Yacht.new(yacht_params)
+     @yacht = Yacht.new(yacht_params)
     if @yachts.save
-      redirect_to @yacht
-    else
-      render :new
-    end
+      redirect_to yacht_path(@yacht)
   end
 end
+end
```

Comment:
extra end

Reply:
No extra `end` here, but careful at the indentation 👍